.github/workflows/bandit.yml

```diff
@@ -0,0 +1,37 @@
+name: Bandit
+
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - 'src/bindings/python/**'
+      - 'samples/python/**'
+      - 'tools/**'
+      - 'scripts/utils/**'
+      - 'docs/**'
+      - '.ci/azure/**'
+  pull_request:
+    paths:
+      - 'src/bindings/python/**'
+      - 'samples/python/**'
+      - 'tools/**'
+      - 'scripts/utils/**'
+      - 'docs/**'
+      - '.ci/azure/**'
```

Comment:
Should we exclude folders in single place in bandit config file and run this action on any *.py file? (like ./**/*.py)

Reply:
Changed this in last commit to run Bandit only when python scripts are updated